internal/k8s/appprotect/app_protect_resources.go

```diff
@@ -148,3 +188,18 @@ func validateAppProtectUserSig(userSig *unstructured.Unstructured) error {
 func GetNsName(obj *unstructured.Unstructured) string {
 	return obj.GetNamespace() + "/" + obj.GetName()
 }
+
+func checkForExtRefs(policy *unstructured.Unstructured) ([]string, error) {
```

Comment:
Hi @rafwegv I think it makes sense to extend the unit tests to test this new function, what do you think?

Reply:
Hi @lucacome the deprecation is only true for KIC. I corrected the doc to reflect that. also added unit test as sugested.